en/components/grid_virtualization.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Grid Virtualization and Performance
+_description: The Ignite UI for Angular Data Grid control now virtualizes its columns and rows, visualizing only the visible chunks of the data in the DOM, as a result allowing smooth and seamless scrolling through large sets of data.
+_keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI widgets, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components Library, Angular Data Grid component, Angular Data Grid control, Angular Grid component, Angular Grid control, Angular High Performance Grid, Virtualization, Performance
+---
+
+## Grid Virtualization and Performance
+
+<p class="highlight">
+    Many modern applications require displaying big data in a way that is performant and seamless to the end user.
```

Comment:
@MayaKirova In this section we need to explain what virtualization is and how we use it in order to boost performance of the grid. Let me know if you want me to write this section.

Reply:
You can add a second paragraph with the information you deem neccessary.